lib/vmdb/loggers/azure_logger.rb

```diff
@@ -0,0 +1,22 @@
+module Vmdb::Loggers
+  class AzureLogger < VMDBLogger
+    def initialize(*loggers)
+      super
+
+      @datetime_format = '%Y-%m-%d %H:%M:%S'
```

Comment:
Why an alternate datetime_format?  Doesn't that make the log messages inconsistent with other VMDBLogger instances?

Reply:
Ok, I'll remove it.

Reply:
If for some reason you don't want `my_logger.datetime_format` to return `nil, you can use the string I provided in the following comment:

https://github.com/ManageIQ/manageiq/pull/17228#discussion_r178615951

Which is what is defaulted in ruby:

https://github.com/ruby/ruby/blob/3be45505/lib/logger.rb#L603-L605

But it does a poor job of making it a constant... which makes me a bit :rage:y